libraries/adaptive-expressions/src/expression.ts

```diff
@@ -206,14 +206,20 @@ export class Expression {
         return { path, refs };
     }
 
+    /**
+     * Parse an expression string into an expression object.
+     * @param expression Expression string.
+     * @param lookup Optional. Function lookup when parsing the expression. Default is Expression.lookup which uses Expression.functions table.
+     * @returns The expression object.
+     */
     public static parse(expression: string, lookup?: EvaluatorLookup): Expression {
         return new ExpressionParser(lookup || Expression.lookup).parse(expression);
     }
 
     /**
      * Lookup a ExpressionEvaluator (function) by name.
-     * @param functionName name of function to lookup
-     * @returns a ExpressionEvaluator that corresponding to the funtion name
+     * @param functionName name of function to lookup.
+     * @returns an ExpressionEvaluator corresponding to the funtion name.
```

Comment:
```suggestion
     * @returns An ExpressionEvaluator corresponding to the function name.
```

Besides the typo, "an" could be capitalized to be consistent with the rest of the changes.
Also _ExpressionEvaluator_  could be put inside a codeblock, although  this would require to update a lot of pre-existent documentation instances.

Reply:
Feedback applied. Thanks!